packages/token-introspection/src/openapi/schemas.yaml

```diff
@@ -1 +0,0 @@
-../../../open-payments/src/openapi/schemas.yaml
```

Comment:
To be a self-contained package, `token-introspection` (and similarly, `open-payments`) should fetch its own schemas